interop-cats/shared/src/test/scala/zio/interop/catzClockSpec.scala

```diff
@@ -0,0 +1,34 @@
+import java.util.concurrent.TimeUnit
+
+import cats.effect.Timer
+import fs2.Stream
+import zio._
+import zio.clock._
+import zio.duration._
+import zio.interop.catz._
+import zio.test.environment._
+import zio.test._
+
+import scala.concurrent.duration.FiniteDuration
+
+object catzClockSpec extends DefaultRunnableSpec {
+
+  def spec =
+    suite("catzClockSpec") {
+      testM("Timer can be constructed from ZIO Clock") {
```

Comment:
This test relies on someone noticing that it ran for 20 seconds to determine whether it failed - would it be appropriate to use a `timeout` aspect here, or is there a more elegant way to make sure that the TestClock.adjust was able to skip over the fs2.Stream.delayBy?

Reply:
@gannicottb Good point! Added a timeout so the test will fail if we aren't actually using the `TestClock`.